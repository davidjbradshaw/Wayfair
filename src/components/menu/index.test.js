import Menu from './index'

test('renders Menu', () => {
  expect(<Menu />).toMatchSnapshot()

  // TODO add open and close tests for Menu
})
