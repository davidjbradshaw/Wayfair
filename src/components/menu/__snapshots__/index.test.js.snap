// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Menu 1`] = `<SimpleMenu />`;
